storage/src/main/java/org/apache/kafka/storage/internals/util/CleanShutdownFileHandler.java

```diff
@@ -85,6 +85,7 @@ void write(long brokerEpoch, int version) throws Exception {
         }
     }
 
+    @SuppressWarnings("unchecked")
     public long read() {
         long brokerEpoch = -1L;
```

Comment:
A few comments unrelated to this PR:
1. Should we return `OptionalInt` to make it clear that it can be unset?
2. Should we use a class instead of map to avoid the need to map it back and forth?

Reply:
Thanks for the advice, updated.